corehq/pillows/blobs.py

```diff
@@ -0,0 +1,26 @@
+from couchexport.models import SavedBasicExport
+from pillowtop.listener import BasicPillow
+
+
+class BlobDeletionPillow(BasicPillow):
+
+    include_docs = False
+    doc_types = {cls.__name__: cls for cls in [
+        # list of all model classes that use BlobMixin
+        SavedBasicExport,
+    ]}
+
+    def change_trigger(self, changes_dict):
+        if not changes_dict.get('deleted', False):
+            return None
+        doc = self.db.get(changes_dict['id'])
```

Comment:
`self.document_store` would be preferable, however I don't see how this call would ever succeed given that the DB is telling us the doc has just been deleted. I think we have to do the same thing UCR does and blindly try to clean up after every deleted document since we won't be able to infer the type after it's deleted.

That might be an argument for addressing this in a `post_delete` signal type of way instead of in a pillow at all